components/builder-web/app/origin/origin-page/origin-keys-tab/origin-keys-tab.component.html

```diff
@@ -33,6 +33,7 @@ <h3 class="actions">Actions</h3>
         Upload a private key
       </button>
     </section>
+    <hr>
     <section class="public">
```

Comment:
For views with more than one section/list of data, I've added a line to better separate them visually.